src/SprykerEco/Yves/Adyen/Controller/NotificationController.php

```diff
@@ -16,7 +16,14 @@
  */
 class NotificationController extends AbstractController
 {
+    /**
+     * @var array
```

Comment:
I guess those can be `array<string, string>` then?

Reply:
Fixed